SeattleColleges/belindas-closet-nextjs
# belinda-nextjs-03-231-modify-add-components-product-page

Resolves #231 
Related #230 

This PR will:
- Refactor the styling using MUI
- Delete the CSS file
- Add a complex Grid to display each product as a Product Card

To test:
- Ensure the front end and back end are running
- Ensure the database has product data
- Log in using an admin, creator, or user account
 - _Note_: Only admin/creator will see delete/archive buttons.
- Open the browser with this link: http://localhost:8080/
- Select any category

Reference:
- [https://mui.com/material-ui/react-grid/#complex-grid](https://mui.com/material-ui/react-grid/#complex-grid)

Screenshot:
User's View:
![image](https://github.com/SeattleColleges/belindas-closet-nextjs/assets/72054441/5996a152-e400-46ab-b369-58a8a7074704)

Admin/Creator's View:
![image](https://github.com/SeattleColleges/belindas-closet-nextjs/assets/72054441/15fade8f-ba4d-46c9-b09a-8f6933a08b7c)
